tests/test_mpi.py

```diff
@@ -1005,6 +1008,182 @@ def test_avoid_haloupdate_if_distr_but_sequential(self, mode):
         calls = FindNodes(Call).visit(op)
         assert len(calls) == 0
 
+    @pytest.mark.parallel(mode=1)
+    def test_issue_2448(self, mode):
+        shape = (2,)
+        so = 2
+
+        grid = Grid(shape=shape)
+
+        # Time related
+        tn = 30
+
+        # Velocity and pressure fields
+        v = TimeFunction(name='v', grid=grid, space_order=so)
+        tau = TimeFunction(name='tau', grid=grid, space_order=so)
+
+        # First order elastic-like dependencies equations
+        pde_v = v.dt - (tau.dx)
+        pde_tau = tau.dt - ((v.forward).dx)
+        u_v = Eq(v.forward, solve(pde_v, v.forward))
+
+        u_tau = Eq(tau.forward, solve(pde_tau, tau.forward))
+
+        # Test two variants of receiver interpolation
+        nrec = 1
+        rec = SparseTimeFunction(name="rec", grid=grid, npoint=nrec, nt=tn)
```

Comment:
Nitpick: You could remove some lines by having `npoint=1, nt=30` in here. You can do similar stuff throughout this test (for example `Grid(shape=(2,))`.